test/verify/check-connection

```diff
@@ -1055,13 +1056,14 @@ until pgrep -f '^(/usr/[^ ]+/[^ /]*python[^ /]* )?/usr/bin/cockpit-bridge'; do s
         self.assertIn("HTTP/1.1 200 OK", m.execute("curl --silent --head http://172.27.0.15:9091"))
 
     @testlib.nondestructive
+    @testlib.skipWsContainer("cockpit/ws doesn't use systemd units")
```

Comment:
This seems wrong? The test worked fine on CoreOS but now the whole test is skipped

Reply:
Ack -- I reworked this.